frontend/src/breeding_form.tsx

```diff
@@ -41,13 +48,13 @@ const useStyles = makeStyles({
 });
 
 const emptyBreeding: Breeding = {
-  breed_date: "",
+  breed_date: null,
   breed_notes: "",
   father: "",
   mother: "",
-  birth_date: "",
+  birth_date: null,
   birth_notes: "",
-  litter_size: 0,
+  litter_size: null,
```

Comment:
What about 0?

Reply:
I want the input field empty when the page renders. It's only empty if the value is null, otherwise it shows a 0.